Makefile.am

```diff
@@ -558,7 +580,9 @@ endif
 
 # Create tss user and FAPI directories directly after installation (vs. after a reboot)
 install-exec-hook:
-	systemd-sysusers && systemd-tmpfiles --create || true
+	(systemd-sysusers && systemd-tmpfiles --create) || \
+	(($(call make_tss_user_and_group) || true) && \
```

Comment:
If make_tss_user_and_group fails, then make_fapi_dirs will also fail, since the chown fails.

IMHO a single || true on the outest would be better.

Reply:
Like this?
```bash
	(systemd-sysusers && systemd-tmpfiles --create) || \
	($(call make_tss_user_and_group) && \
	$(call make_fapi_dirs)) || true
```
This would not trigger FAPI folder creation in case user and group creation fails. Is that intended? Because I think, the folders should be created even if user and group creation fails. This way, at least root user can use FAPI.

Reply:
Are we sure that we don't intend for this behavior ?
If not, however, I'd split and rearrange things a little:
`make_fapi_dirs && make_tss_user_and_group && chown_fapi_dirs || true`